python/api-examples-source/mpa-hello/requirements.txt

```diff
@@ -1,9 +1,4 @@
 pandas==1.5.3
```

Comment:
These requirements other than `opencv-python-headless` are installed as `streamlit`'s deps, so not necessary actually, but specifying them here is a good idea to explicitly shows the dependencies.